<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="GGKU4DELL1188" ignored="4" name="Test" tests="1" failures="1" timestamp="2021-02-02T18:34:06 IST" time="0.04" errors="0">
  <testcase name="test1" time="0.009" classname="com.test.selenium.ReverseString">
    <failure type="java.lang.AssertionError" message="deliberately failed">
      <![CDATA[java.lang.AssertionError: deliberately failed
at org.junit.Assert.fail(Assert.java:88)
at com.test.selenium.ReverseString.test1(ReverseString.java:59)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="test4" time="0.0" classname="com.test.selenium.ReverseString">
    <ignored/>
  </testcase> <!-- test4 -->
  <testcase name="test2" time="0.0" classname="com.test.selenium.ReverseString">
    <ignored/>
  </testcase> <!-- test2 -->
  <testcase name="test" time="0.0" classname="com.test.selenium.ReverseString">
    <ignored/>
  </testcase> <!-- test -->
  <testcase name="test3" time="0.0" classname="com.test.selenium.ReverseString">
    <ignored/>
  </testcase> <!-- test3 -->
</testsuite> <!-- Test -->
